grakn-graph/src/main/java/ai/grakn/graph/internal/ResourceImpl.java

```diff
@@ -71,9 +73,15 @@
      */
     @Override
     public Collection<Thing> ownerInstances() {
-        return getShortcutNeighbours().stream().
+        //Get Owner via implicit structure
+        Set<Thing> owners = getShortcutNeighbours().stream().
```

Comment:
Good suggestion. Will think about doing that in a future cleanup PR. May break stuff all over again.